Dockerfile.compose

```diff
@@ -2,37 +2,20 @@
 
 FROM debian:stretch-slim
 
-RUN apt-get -y update
+# Install Dependencies
+RUN apt-get update && apt-get install -y \
+    python3 \
+    python3-pip \
+ && rm -rf /var/lib/apt/lists/*
 
-# Install Python
-RUN apt-get -y install python3
-RUN apt-get -y install apt-transport-https
-RUN apt-get -y install curl
-RUN apt-get -y install gnupg2
-RUN apt-get -y install git
+# Install python dependencies. Do this before copying the entire project to keep cache.
+COPY requirements.txt /tmp/
+RUN pip3 install --requirement /tmp/requirements.txt
 
-# Install Tesla API Scraper
-RUN apt-get -y install python3-pip
-WORKDIR /
-RUN pip3 install influxdb
-
-RUN git clone https://github.com/tkrajina/srtm.py
-WORKDIR srtm.py
-RUN python3 ./setup.py install
-WORKDIR /
-
-ARG CACHEBUST=1
-ARG gitversion
-# Configure it
-RUN git clone https://github.com/lephisto/tesla-apiscraper
-WORKDIR tesla-apiscraper
-RUN git pull
-RUN git checkout $gitversion
-# Define our startup script
-RUN echo "#!/bin/bash" > /start.sh
-RUN echo "python3 /tesla-apiscraper/apiscraper.py" >> /start.sh
-RUN chmod +x /start.sh
+# Copy in this project
+COPY . /tesla-apiscraper
```

Comment:
In the most usual rebuild workflow, only `.py` files in the local workdir will have changed. In that case, this is the first line of the rebuild, which is *very* fast.